extensions/sql-database-projects/src/test/project.test.ts

```diff
@@ -192,21 +192,21 @@ describe('Project: sqlproj content operations', function (): void {
 		const project = await Project.openProject(projFilePath);
 
 		should(project.databaseReferences.length).equal(0, 'There should be no datbase references to start with');
-		await project.addSystemDatabaseReference(SystemDatabase.master);
+		await project.addSystemDatabaseReference({ databaseName: 'master', systemDb: SystemDatabase.master });
 		should(project.databaseReferences.length).equal(1, 'There should be one database reference after adding a reference to master');
 		should(project.databaseReferences[0].databaseName).equal(constants.master, 'The database reference should be master');
 		// make sure reference to SSDT master dacpac was added
 		let projFileText = (await fs.readFile(projFilePath)).toString();
 		should(projFileText).containEql(convertSlashesForSqlProj(project.getSystemDacpacSsdtUri(constants.master).fsPath.substring(1)));
 
-		await project.addSystemDatabaseReference(SystemDatabase.msdb);
+		await project.addSystemDatabaseReference({ databaseName: 'msdb', systemDb: SystemDatabase.msdb });
 		should(project.databaseReferences.length).equal(2, 'There should be two database references after adding a reference to msdb');
 		should(project.databaseReferences[1].databaseName).equal(constants.msdb, 'The database reference should be msdb');
 		// make sure reference to SSDT msdb dacpac was added
 		projFileText = (await fs.readFile(projFilePath)).toString();
 		should(projFileText).containEql(convertSlashesForSqlProj(project.getSystemDacpacSsdtUri(constants.msdb).fsPath.substring(1)));
 
-		await project.addDatabaseReference(Uri.parse('test.dacpac'), DatabaseReferenceLocation.sameDatabase);
+		await project.addDatabaseReference({ dacpacFileLocation: Uri.file('test.dacpac'), databaseLocation: DatabaseReferenceLocation.sameDatabase });
```

Comment:
the master and msdb references confirm that the entry's correct in the sqlproj file; is there a way to do that here as well, but for dacpacs? (or is that covered by another test?)

Reply:
those checks above were just meant to verify that an entry was also added for the SSDT location of the system dacpacs, but now I added checks to verify that the ADS system dacpacs and the test.dacpac got added to the sqlproj too

Reply:
Perfect; thanks!